lib/python/vdm/server/HTTPListener.py

```diff
@@ -1123,7 +1123,7 @@ def put(database_id):
             return response
         except Exception, err:
             print traceback.format_exc()
-            return make_response(jsonify({'statusstring': str(err)}),
+            return make_response(jsonify({'statusString': str(err)}),
```

Comment:
If there is an unexpected error, status should not be 200 (OK). May be use 500 and make the http status code the same as well.